server/apps/ldap/ldap.py

```diff
@@ -218,8 +221,18 @@ def add_default_values(doc, user_name, user_type, **kwargs):
     Adds user_name, user_type, is_active: True, needs_activation: False and the values passed to **kwargs to doc.
     """
 
-    doc['username'] = user_name
+    doc['username'] = user_name.strip()
     doc['user_type'] = 'user' if user_type is None else user_type
     doc['is_active'] = True
     doc['needs_activation'] = False
     doc.update(**kwargs)
+
+
+def get_user_query(username):
+    """
+    Get the user query.
+    :param {str} username:
+    :return {dict}: query
```

Comment:
In Sphinx-style docstrings, parameter type names should not be enclosed in curly braces, plain `str` and `dict` already do the job.